core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/NativeImageResourcePatternsBuildItem.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.deployment.builditem.nativeimage;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import io.quarkus.deployment.util.GlobUtil;
+
+/**
+ * A build item that indicates that a set of resource paths defined by regular expression patterns or globs should be
```

Comment:
Maybe it wouldn't hurt to explain what `globs` means...

Reply:
Yeah, let me do that. I should also add some JavaDoc to the builder methods.